docs/walkthrough.rst

```diff
@@ -0,0 +1,199 @@
+..
+
+We added several columns to annotate our dataset with Key-Value pairs \
+following the `REMBI <https://doi.org/10.1038/s41592-021-01166-8>`_ guidelines \
+(after saving the document, our ``TAB`` separator became ``,``).
+
+We proceed and start the script "*Import from CSV*".
+
+.. figure:: images/annotation_06_script_import.png
+
+   *Selection of all the datasets of a project and the Import from CSV script.*
+..
+
+There are many parameters here that could help you fine tune the way to import annotations. More \
+  on that in the :doc:`parameter description section</parameter_description>` of this guide.
+
+
+We can then see in the OMERO activities that the Key-Value pairs were added to 5 \
+datasets out of the 11 present in this project (as expected). We also see three set of \
+key-value pairs, one for each namespace we used in the csv.
+
+.. figure:: images/annotation_07_KV_import_printout.png
+
+  *The script output (5 entries in the csv matched to 5 dataset out of 11) and the\
+  resulting key-value pairs annotation.*
+..
+
+Converting the Key-Value pairs namespace
+----------------------------------------
+
+Key-Value pairs are assigned a category/label (known as namespace). \
+This grants flexibility so that different annotations can be \
+distinguished or isolated (like for exporting/deleting only those with a given \
+namespace).
+
+   In fact, if you created Key-Value pairs in OMERO.web, you have used \
+   namespaces without noticing it: OMERO assigns by default the \
+   "client namespace" (``openmicroscopy.org/omero/client/mapAnnotation`` in full)\
+   , a special namespace recognized by OMERO.web allowing to edit them.
+
+We would like to do a few changes in OMERO.web, and for that we would need to \
+change the namespace to the default OMERO.web namespace. We indicate the namespace we \
+want to edit and leave the second field empty (default namespace is then used).
+
+.. figure:: images/annotation_08_convert_namespace.png
+
+  *The script to convert the namespace of key-value pairs annotations.*
+..
+
+And here is our five Key-Value pairs annotations with converted namespace:
+
+.. figure:: images/annotation_09_converted_KV.png
+
+  *The script output (5 dataset had annotations with the default namespace) and the\
+  resulting key-value pairs annotation.*
+..
+
+Deleting Key-Value pairs
+------------------------
+
+It turns out that there are more than a few edits to do on our key-value pairs, and \
+converting back and forth the namespace seems too tedious this time. Let's instead export \
+what we have so far (we have done edits that we don't want to lose), delete "old" the key-value pairs, \
+and reimport when we are done fixing them.
+
+Using the same selection inputs for both scripts, we make sure that we have the key-value pairs on our \
+computer before deleting them. Because we want to export/delete multiple namespace at once, we can use the ``*`` \
+to include all namespace.
```

Comment:
```suggestion
to include all namespaces.
```

But mainly, this ^^^ sounds scary and needs an additional sentence. Again, protect the user from the ``namespace`` term, and, also, in time where big deletions are asked for, make sure to impart sense of security by explaining slowly and properly, ta.